tests/installer/checksums.sh

```diff
@@ -1,19 +1,49 @@
 #!/bin/bash
-
+#
+# Mechanism to validate kickstart files integrity status
+#
+# Copyright: SPDX-License-Identifier: GPL-3.0-or-later
+#
+# Author  : Pawel Krupa (pawel@netdata.cloud)
+# Author  : Pavlos Emm. Katsoulakis (paul@netdata.cloud)
 set -e
+README_DOC="packaging/installer/README.md"
 
-if [ ! -f .gitignore ]; then
-	echo "Run as ./tests/installer/$(basename "$0") from top level directory of git repository"
-	exit 1
+# If we are not in netdata git repo, at the top level directory, fail
+TOP_LEVEL=$(basename "$(git rev-parse --show-toplevel 2> /dev/null || echo "")")
+CWD="$(git rev-parse --show-cdup 2> /dev/null || echo "")"
+if [ ! -z $CWD ] || [ ! "${TOP_LEVEL}" == "netdata" ]; then
+    echo "Run as .travis/$(basename "$0") from top level directory of netdata git repository"
+    echo "Kickstart validation process aborted"
+    exit 1
 fi
 
 for file in kickstart.sh kickstart-static64.sh; do
-	OLD_CHECKSUM=$(grep "$file" packaging/installer/README.md | grep md5sum | cut -d '"' -f2)
-	NEW_CHECKSUM="$(md5sum "packaging/installer/$file" | cut -d' ' -f1)"
-	if [ "$OLD_CHECKSUM" != "$NEW_CHECKSUM" ]; then
-		echo "Invalid checksum for $file in docs."
-		echo "checksum in docs: $OLD_CHECKSUM"
-		echo "current checksum: $NEW_CHECKSUM"
-		exit 1
+	CHECKSUM_IN_README=$(grep "$file" $README_DOC | grep md5sum | cut -d '"' -f2)
```

Comment:
I'd name this var README_MD5, for consistency.

Reply:
ack